server/mockapi/mockapi.feature

```diff
@@ -0,0 +1,17 @@
+# Copyright Strimzi authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+Feature: mockapi module
+
+    Behaviours and capabilities provided by the mockapi module
+
+    Scenario: Returns with the expected response for a mocked api call
```

Comment:
Is this for test purposes? Or just there until we have a full API module?

Reply:
It is placeholder, pending details of `strimzi-admin` (and thus what we have to mock)